mode/vue/index.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
```

Comment:
This appears to be a copy-pasted versin of the HTML mixed mode demo. That doesn't really show what this mode can do, does it?